app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -328,8 +329,17 @@ MinionPoller = {
   },
 
   alertFailedBootstrap: function() {
-    if (!$('.failed-bootstrap-alert').length) {
-      showAlert('At least one of the nodes is in a failed state. Please run "supportconfig" on the failed node(s) to gather the logs.', 'alert', 'failed-bootstrap-alert');
+    var cachedFailedLastOrchestration = window.localStorage.getItem('failedLastOrchestrationAt');
+
+    if (!$('.failed-bootstrap-alert').length &&
+        cachedFailedLastOrchestration !== State.lastOrchestrationAt) {
+      var $alert = showAlert('At least one of the nodes is in a failed state. Please run "supportconfig" on the failed node(s) to gather the logs.', 'alert', 'failed-bootstrap-alert');
+
+      window.localStorage.removeItem('failedLastOrchestrationAt');
+      $alert.on('closed.bs.alert', function () {
```

Comment:
This isn't a request for change necessarily, but please be aware that "bs" has a very specific meaning in English vernacular, and it's neither _bootstrap_ nor _build service_. A "bs alert" just makes me snicker ;-)